### PR TITLE
Disable automatic sizing on grimoire bar to allow text to fit

### PR DESCRIPTION
**Pull request by Pidgeot**
As mentioned in #976, the Steam version still has an issue causing the grimoire bar to appear too small to fit the extra text.

As best as I can tell (and this might not be correct, as I don't have a save handy to test the web version), this might have been a bugfix in the Steam version (2.04): the minigame's draw function was *always* setting it like that, but I think it might not have worked because the existing HTML template didn't have a style element.

Rather than trying to guess a different way to scale the bar, this change simply removes the width style entirely after the bar is drawn, making it stretch across the full width of the containing element like this:

image

### CODE REVIEW

**Comment by DanielNoord** [comment]
I saw that Orteil just released a beta version for the Steam version, could you see if the issue is resolved here?
I don't quite like stretching the bar to the full width of the container and would like to make it as Orteil intended. In the browser version it seems to be set to something around 75% of the container-width. If the beta doesn't solve this issue I would ask you to set the width to that.

**Comment by Pidgeot** [comment]
The beta does not resolve the issue, so per your request, I've modified it to set the width to 75% instead:

image

File: src/Main/ReplaceGameElements/NativeGrimoire.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ function ReplaceNativeGrimoireDraw() {
         minigame.magicBarTextL.innerHTML += ` (${FormatTime(
           CalculateGrimoireRefillTime(minigame.magic, minigame.magicM, minigame.magicM),
         )})`;
+        minigame.magicBarL.style.width = "75%";
```
Comment (DanielNoord) [inline src/Main/ReplaceGameElements/NativeGrimoire.js]:
```suggestion
        minigame.magicBarL.style.width = "75%" // TODO: Fix this after Orteil pushes fix to main game;
```

**Comment by DanielNoord** [comment]
Thanks! Really appreciated!

**Pull request merged**